churchland_pipeline/tasks/pacman.py

```diff
@@ -81,28 +102,55 @@ class Trial(dj.Part):
         reward : longblob # reward delivery
         photobox : longblob # photobox signal
         """
-    
+
+# -------------------------------------------------------------------------------------------------------------------------------
+# LEVEL 1
+# -------------------------------------------------------------------------------------------------------------------------------
+
+@schema
+class TrialAlignment(dj.Lookup):
+    definition = """
+    # Task state IDs used to align trials
+    -> TaskState
+    ---
+    """
+
+# -------------------------------------------------------------------------------------------------------------------------------
+# LEVEL 2
+# -------------------------------------------------------------------------------------------------------------------------------
+
 @schema
-class TrialAlignment(dj.Imported):
+class AlignmentIndices(dj.Imported):
     definition = """
+    -> TrialAlignment
     -> acquisition.SyncChannel
     trial_number : smallint unsigned # trial number (within session)
     ---
     alignment_index : int unsigned # alignment index (in Speedgoat time base)
     speedgoat_alignment : longblob # alignment indices for Speedgoat data
     ephys_alignment : longblob # alignment indices for Ephys data
-    """    
-
-# -------------------------------------------------------------------------------------------------------------------------------
-# LEVEL 1
+    """   
+    
 # -------------------------------------------------------------------------------------------------------------------------------
+# LEVEL 3
+# -------------------------------------------------------------------------------------------------------------------------------    
+@schema
+class Emg(dj.Imported):
+    definition = """
+    # raw, trialized, and aligned EMG data
+    -> acquisition.EmgChannelGroup
+    -> AlignmentIndices
+    emg_channel : tinyint unsigned # channel number (indexed relative to EMG channel group)
+    ---
+    emg_channel_data : longblob # channel data
```

Comment:
`_data` is rather vague.   What exactly is being stored? E.g. `emg_voltage_signal: longblob # (mV)`